mkosi.uki-profiles/90-factory-reset.conf

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[UKIProfile]
+Profile=
+        ID=factory-reset
+        TITLE=Reset System to Factory Defaults [CAUTION!]
+
+Cmdline=
+        systemd.factory_reset=1
+
+SignExpectedPcr=yes
```

Comment:
Not sure if I like this, since this overrides the kernel command line from the base .cmdline section. Either the contents of `KernelCommandLine=` in mkosi.conf should be replicated here, or we need a way to append to the base kernel command line here.